README.md

```diff
@@ -1,24 +1,26 @@
 [![Licensed under the MIT License](https://img.shields.io/badge/License-MIT-blue.svg)](https://github.com/microsoft/spatialaudio-unity/blob/master/LICENSE)
 
 # Introduction 
-This repository provides plugins and tools for integrating spatial audio and acoustics into your Unity 3D applications and games. This includes:
-- A HoloLens 2 spatializer plugin which uses [Windows Sonic API](https://docs.microsoft.com/en-us/windows/win32/coreaudio/spatial-sound) to enable hardware offload of spatial audio processing, freeing up the CPU for your application.
-- A sample Unity application that demonstrates proper usage of the HoloLens 2 spatializer plugin.
-- Source code for the [Project Acoustics](http://aka.ms/acoustics) spatializer plugin.
+This repository provides plugins and tools for integrating spatial audio and acoustics into your Unity 3D applications and games  
+- A **cross-platform spatializer plugin (supported on Windows and Android)** that uses FLEX DSP algorithms for highly efficient spatial audio processing.
+- A sample Unity application that demonstrates cross-platform spatializer plugin configuration and usage.
 
-# Getting started with Spatial Audio for Unity on HoloLens 2
+# Getting started with Spatial Audio for Unity
 Cloning this repository is not required to start using the Microsoft Spatializer in your Unity project. Visit our [documentation](https://docs.microsoft.com/en-us/windows/mixed-reality/spatial-sound-in-unity) for instructions on integrating the Microsoft Spatializer into your Unity project. For a more in-depth exploration of spatial audio, check out the [learning module](https://docs.microsoft.com/en-us/windows/mixed-reality/unity-spatial-audio-ch1). If you'd like to build the plugin yourself, see below.

```

Comment:
It would suggest updating this to avoid using possessive language.  "Visit the documentation site for instructions...."